os/arch/arm/src/amebad/amebad_i2c.h

```diff
@@ -0,0 +1,104 @@
+/****************************************************************************
```

Comment:
Please check defined functions's validation, it seems that nowhere using them now.